tests/testthat/test-object_length_linter.R

```diff
@@ -43,7 +43,22 @@ test_that("lints S3 generics correctly", {
 
 test_that("object_length_linter won't fail if an imported namespace is unavailable", {
   expect_length(
-    lint_package(test_path("dummy_packages", "missing_dep"), linters = object_length_linter(), parse_settings = FALSE),
+    lint_package(
+      test_path("dummy_packages", "missing_dep"),
+      linters = object_length_linter(),
```

Comment:
the original issue is about object_usage_linter, right?

Reply:
No, it's about `object_length_linter()`: https://github.com/r-lib/lintr/issues/1503